TestControllers/AzureController.psm1

```diff
@@ -233,6 +240,9 @@ Class AzureController : TestController
 
 		Write-LogInfo "Setting global variables"
 		$this.SetGlobalVariables()
+		if (!$global:AllTestVMSizes) {
```

Comment:
Move this to function SetGlobalVariables?

Reply:
This is only for Azure Platform usage, and could not be assigned multi times, so just place at the end of Prepare Environment, in stead of  wrapped in SetGlobalVariable function.

Reply:
Oh, I see the SetGlobalVariables is not defined for Azure. Then we can add that function in AzureController, with overwritten behavior

Reply:
it's only be defined once, and should only be initialize once for Azure platform, and should be initialized before LoadTestCases,  so,  leave this code in PrepareEnvironment.  If put in SetGlobalVariables(), it's value should not be initialized and changed again, especially in case:  this SetGlobalVariables() can be invoked by others several times.

Reply:
@lizzha   Moved to SetGlobalVariables()  but always 

if (!$global:AllTestVMSizes) {}